src/main/java/hwr/oop/group4/todo/persistence/SavePersistenceAdapter.java

```diff
@@ -1,8 +1,8 @@
 package hwr.oop.group4.todo.persistence;
 
-import java.io.File;
+import hwr.oop.group4.todo.persistence.configuration.FileAdapterConfiguration;
 
 public interface SavePersistenceAdapter {
 
-    void save(Persistable data, File file);
+    void save(Persistable data, FileAdapterConfiguration config);
```

Comment:
shoudn't it be Persitable data, SaveConfiguration config

Reply:
The configuration would be the same for both load and save. So I guess one configuration for the FileAdapter would be enough. What do you think?

Reply:
You can use one configuration for both